test/Autofac.Test.Compilation/Autofac.Test.Compilation.csproj

```diff
@@ -26,8 +26,11 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Compile Remove="Helpers\**" />
```

Comment:
It looks like the generator project used to be in here but maybe moved and resulted in some cruft showing up?

I admit, I had to go digging in git history to remember that `Autofac.Test.Compilation` was mostly about nullable references than about _compiling Autofac_ so maybe some clarification here might help at some point in the form of README files. I was about to suggest we combine the two since they're sort of both about compilation but I do understand they're about the _consumer of Autofac_ compiling vs _Autofac itself_ compiling so having them split also makes some sense.

Reply:
I did indeed initially add them to this test project, then realised that the two projects were coming at it from completely different ends, as you say.